test/test_cursor.py

```diff
@@ -1133,6 +1133,12 @@ def test_tailable(self):
 
         self.assertEqual(3, db.test.count())
 
+        # __getitem__(index)
+        cursor2 = db.test.find(cursor_type=CursorType.TAILABLE)
+        self.assertEqual(4, cursor2[0]["x"])
+        self.assertEqual(5, cursor2[1]["x"])
+        self.assertEqual(6, cursor2[2]["x"])
+
```

Comment:
Can test this with `CursorType.TAILABLE_AWAIT` too? And another for `__getitem__(slice)`, eg
```python
cursor = db.test.find(cursor_type=CursorType.TAILABLE_AWAIT)
self.assertEqual([4,5], [doc["x"] for doc in cursor[0:2]])
```

Reply:
@ShaneHarvey Thanks for your advice. I will expand the tests later.